lib/book.rb

```diff
@@ -0,0 +1,37 @@
+class Book
+  attr_reader :book,
+              :reviews
+  attr_writer :read
+
+  def initialize(book)
+    @book = book
+    @read = false
+    @reviews = []
+  end
+
+  def title
+    @book.values.first
```

Comment:
This presumes that the values method will return things in order, which is not guaranteed.